packages/react/src/runtimes/edge/streams/toolResultStream.ts

```
@@ -64,38 +64,38 @@ export function toolResultStream(
+                  toolCallType,
+                  toolCallId,
+                  toolName,
+                  result,
+                });
+              } catch (error) {
+                controller.enqueue({
+                  type: "tool-result",
+                  toolCallType,
+                  toolCallId,
+                  toolName,
+                  result: "Error: " + error,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)